File: auto-docs/executables/r/privacy_true.r
library(plotly)
py <- plotly(username='TestBot', key='r1neazxo9w')

data <- list(
  list(
    x = c(0, 2, 4), 
    y = c(0, 4, 2), 
    type = "scatter"
  )
)
response <- py$plotly(data, kwargs=list(filename="privacy-true", world_readable=TRUE, fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/nodejs/privacy_true.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [0, 2, 4], 
    y: [0, 4, 2], 
    type: "scatter"
  }
];
var graph_options = {filename: "privacy-true", world_readable: true, fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});